packages/components/src/components/AssetLogo/AssetLogo.tsx

```diff
@@ -61,6 +61,7 @@ export const AssetLogo = ({
     const [isPlaceholder, setIsPlaceholder] = useState(!shouldTryToFetch);
     const fileName = contractAddress ? `${coingeckoId}--${contractAddress}` : coingeckoId;
     const logoUrl = getAssetLogoUrl(fileName);
+    const placeholderUppercase = placeholder.toUpperCase();
```

Comment:
Please do not modify it here. It changes behaviour on 5 places in the codebase. Uppercase it in `TokenIconSet` directly when passing the prop.

Reply:
I thought we want it to be uppercase in all cases, oki then will modify it for our case only